tests/Unit/Evolution/Systems/Cce/Test_DumpBondiSachsOnWorldtube.cpp

```diff
@@ -235,6 +245,154 @@ void test(const std::string& filename_prefix,
   }
 }
 
+// Case when Klein Gordon scalar variables are also dumped on the worldtube
+void test_kg(const std::string& filename_prefix,
+                const std::vector<double>& radii) {
```

Comment:
Again, instead of copying, just template the original test function on `IncludeKleinGordon` and interject `if constexpr`s where needed.